### DISCUSSION TITLE
My latex code snippets is not working

### DISCUSSION THREAD

**Discussion by mynanshan** [zed-industries/zed#22457] [Feature Requests]
My code snippets defined in .config/zed/snippets/latex.json cannot be triggered. Meanwhile, other snippets, such as python.json, work well. Do you have any ideas on how I can inspect this issue?

I am currently using:
- Zed preview v0.167.1
- Zed latex extension v0.1.4
- TexLab 5.21.0

**Comment by mynanshan** [comment]
@Zed preview v0.174.2.

Now the problem is fixed. My customized snippets work.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
